airesources/OCaml/const.ml

```diff
@@ -0,0 +1,28 @@
+(* OCaml Starter for Halite II on Halite.io
+   This code is public domain. There is no warranty.
+ *)
+
+let ship_radius = 0.5;;
+
+let dock_radius = 4.;;
+
+let max_speed = 7.;;
```

Comment:
I might prefer this to be an int, not float, to make it clear speeds as accepted by game are integers.